Sources/AWSSignerV4/hash.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  hash.swift
+//  AsyncHTTPClient
+//
+//  Created by Adam Fowler on 2019/08/29.
+//
+
+import Foundation
```

Comment:
import detail...

Reply:
This is all going in the bin in the near future, gonna leave it as is just now

Reply:
What do you mean with "in the bin"? Would love to keep this tidy... Just `import Foundation` is a regression imho.

Reply:
All of hash.swift is about to replaced by swift-crypto